CMakeLists.txt

```diff
@@ -247,10 +247,10 @@ if (MRDOX_BUILD_TESTS)
         # test run
         include(mrdox.cmake)
 
-        mrdox(FORMAT adoc CONFIG docs/mrdox.yml SOURCE ${TEST_SOURCE} OUTPUT ${CMAKE_CURRENT_BINARY_DIR}/test-docs)
-        mrdox(FORMAT xml  CONFIG docs/mrdox.yml SOURCE ${TEST_SOURCE} OUTPUT ${CMAKE_CURRENT_BINARY_DIR}/test-docs)
-        mrdox(FORMAT test CONFIG docs/mrdox.yml SOURCE ${TEST_SOURCE} test_generator OUTPUT ${CMAKE_CURRENT_BINARY_DIR}/test-docs
-              PLUGINS_PATH ${CMAKE_CURRENT_BINARY_DIR})
+        mrdox(FORMAT adoc CONFIG docs/mrdox.yml SOURCES ${TEST_SOURCE} OUTPUT ${CMAKE_CURRENT_BINARY_DIR}/test-docs)
+        mrdox(FORMAT xml  CONFIG docs/mrdox.yml SOURCES ${TEST_SOURCE} OUTPUT ${CMAKE_CURRENT_BINARY_DIR}/test-docs)
+        mrdox(FORMAT test CONFIG docs/mrdox.yml SOURCES ${TEST_SOURCE} test_generator
```

Comment:
I don't understand, what does this do?

Reply:
Run mrdox with the test plugin, so we check the plugin system when we build.